examples/nmea2000/fast_packet_protocol/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.16)
+project(fast_packet_protocol_example)
+
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Just a note, I opened #340 because apparently there are nicer or at least more modern ways to do this 🙂